frame/bonded-finance/src/tests.rs

```diff
@@ -92,6 +95,46 @@ proptest! {
 			  })?;
 	  }
 
+	  #[test]
+	  fn generate_vesting_schedule(offer in simple_offer(1)) {
+		ExtBuilder::build().execute_with(|| {
+			let mut offer = offer.clone();
+			offer.beneficiary = BOB;
+			System::set_block_number(1);
+			prop_assert_ok!(Tokens::mint_into(NATIVE_CURRENCY_ID, &ALICE, Stake::get()));
+			prop_assert_ok!(Tokens::mint_into(offer.reward.asset, &ALICE, offer.reward.amount));
+			let offer_id = BondedFinance::do_offer(&ALICE, offer.clone(), false);
+			prop_assert_ok!(offer_id);
+			let offer_id = offer_id.expect("impossible; qed");
+
+			prop_assert_ok!(BondedFinance::do_bond(offer_id, &BOB, 1, false));
+
+			let schedule = VestingSchedule {
+				window: BlockNumberBased { start: 1_u64, period: 1_u64 },
+				 period_count: 1_u32,
+				   per_period: 1000000_u128,
+			};
+
+			println!("{:?}", System::events());
```

Comment:
afaik we do not merge println in tests

Reply:
nope, it was temporarily there while still in draft.